examples/colors.cpp

```diff
@@ -58,6 +58,8 @@ int main() {
                       << "\033[0m";
         }
         std::cout << "\n";
+        std::cout << "Press any key to quit"<<std::endl;
+        std::cin.get();
```

Comment:
We have to remove the example from the CI script when we do that. On Linux I haven't sound that necessary, but for opening the file through the file manager it probably is. But I'd like to make something in C++ Terminal instead that waits for input unless the terminal is not interactive (this will mean that if the executable is run through the CI, it won't wait for any keypress). So yeah I'd remove that for now and make an issue on that for later. Should be a quick fix.

Reply:
Looks fine and a very useful feature